packages/shared/lib/services/connection.service.ts

```diff
@@ -502,16 +502,27 @@ class ConnectionService {
 
     public async listConnections(
         environment_id: number,
-        connectionId?: string
+        connectionId?: string,
+        limit: number | null = 20,
+        offset = 0,
+        integration?: string
```

Comment:
We broke listConnections last time we modified it. Can we add unit tests to make sure we are not changing the behavior and that it works as expected.
Also all the params except  environment_id should be explicitly optional